UI/MapboxNavigationUI/Bundle.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+extension Bundle {
+    class var navigationUI: Bundle {
+        get {
+            let bundle = Bundle(for: RouteViewController.self)
+            let resourceBundlePath = "\(bundle.bundlePath)/MapboxNavigationUI.bundle"
```

Comment:
Resource bundles are generally associated with static libraries. Since this library is distributed as a dynamic framework, would it be possible to put assets directly inside the framework’s resource folder?